libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -12,22 +12,22 @@ import { TurnContext } from './turnContext';
 
 /**
  * Describes a bot activity event handler, for use with an [ActivityHandler](xref:botbuilder-core.ActivityHandler) object.
- * 
+ *
```

Comment:
I think we should not fix these spaces in this PR, it makes the review more difficult and is not related to the issue